src/cluster.c

```diff
@@ -5527,6 +5526,83 @@ void clusterReplyMultiBulkSlots(client * c) {
     setDeferredArrayLen(c, slot_replylen, num_masters);
 }
 
+sds genClusterInfoString(sds info){
```

Comment:
I think this should generate it's own SDS string. Later on you can just do `sdscatsds(info, genClusterInfoString())`;